charts/fint-adapter-sikri/templates/fint-adapter-op-item.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.onePasswordVault.create -}}
+apiVersion: onepassword.com/v1
+kind: OnePasswordItem
+metadata:
+  name: {{ .Values.nameOverride }}
```

Comment:
use `{{ include "fint-adapter-sikri.name" . }}` instead.

maybe it should be suffixed with `-secret`?